read.js

```diff
@@ -0,0 +1,105 @@
+const { MongoClient } = require('mongodb');
+
+async function main() {
+    /**
+     * Connection URI. Update <username>, <password>, and <your-cluster-url> to reflect your cluster.
+     * See http://bit.ly/NodeDocs_lauren for more details
+     */
+    const uri = "mongodb+srv://<username>:<password>@<your-cluster-url>/test?retryWrites=true&w=majority";
+
+    /**
+     * The Mongo Client you will use to interact with your database
+     * See bit.ly/Node_MongoClient for more details
+     */
+    const client = new MongoClient(uri);
+
+    try {
+        // Connect to the MongoDB cluster
+        await client.connect();
+
+        // Make the appropriate DB calls
+
+        // Find the listing named "Infinite Views" that we created in create.js
+        await findOneListingByName(client, "Infinite Views");
+
+        // Find up to 5 listings with at least 4 bedrooms and at least 2 bathrooms
+        // If you recently ran create.js, a listing named Beautiful Beach House should be included in the results 
+        await findListingsWithMinimumBedroomsBathroomsAndMostRecentReviews(client, {
```

Comment:
take or leave it: since you don't need to be explicit with the positional parameters, might want to rename this to something shorter like `findListingsAndFilter`?